File: QID-3447-SFElognormal/SFElognormal.R
# clear variables and close windows
rm(list = ls(all = TRUE))
graphics.off()

# Main computation
par(mfrow = c(1, 1))
margins     = c(1, 1, 1, 1)
normaxis    = seq(from = -5, to = 15, by = 0.1)
lognormaxis = seq(from = 0.01, to = 15, by = 0.1)
n           = dnorm(normaxis)
ln          = dnorm(log(lognormaxis))

# Plot
plot(normaxis, n, col = "blue", type = "l", lty = 1, lwd = 3, xlim = c(-5, 15), ylim = c(0, 
    0.5), main = "Normal and Log Normal distribution", xlab = "X", ylab = "Y")
points(lognormaxis, ln, col = "red", type = "l", lty = 2, lwd = 3)

File: QID-3447-SFElognormal/SFElognormal.m
clear
clc
close all

%% Main computation
normaxis    = -5 : .1 : 15;
lognormaxis = 0.01 : .1 : 15;
n           = normpdf(normaxis);
ln          = normpdf(log(lognormaxis));

%% Plot

hold on

plot(normaxis, n, 'b', 'LineWidth', 2, 'LineStyle', '-');
plot(lognormaxis, ln, 'r', 'LineWidth', 2, 'LineStyle', '--');
axis([-5 15 0 0.5]);
title('Normal and Log Normal distributions'); 
xlabel('X')
ylabel('Y')

hold off

figure;
subplot(1, 2, 1);
plot(normaxis, n, 'b', 'LineWidth', 2, 'LineStyle', '-');
axis([-5 5 0 0.5])
title('Normal distribution'); 
xlabel('X')
ylabel('Y')

subplot(1, 2, 2);
plot(lognormaxis, ln, 'b', 'LineWidth', 2, 'LineStyle', '-');
axis([0 15 0 0.5])
title('Log normal distribution');
xlabel('X')
ylabel('Y')